### PR TITLE
[Admin] Add link to product in variant breadcrumb

### PR DESCRIPTION
**Pull request by Tomanhez**
| Q               | A
| --------------- | -----
| Branch?         | master 
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

**New link:**
Screenshot 2019-07-08 at 10 38 28

**After clicking:**
**to url:**
Screenshot 2019-07-08 at 13 59 57

**Product edit page:**
Screenshot 2019-07-08 at 14 00 05

### CODE REVIEW

File: src/Sylius/Bundle/AdminBundle/Resources/views/ProductVariant/Update/_breadcrumb.html.twig [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {% set breadcrumbs = [
         { label: 'sylius.ui.administration'|trans, url: path('sylius_admin_dashboard') },
         { label: 'sylius.ui.products'|trans, url: path('sylius_admin_product_index') },
-        { label: app.request.attributes.get('productId') },
+        { label: app.request.attributes.get('productId'), url: path('sylius_admin_product_index') ~ app.request.attributes.get('productId') },
```
Comment (GSadee) [inline src/Sylius/Bundle/AdminBundle/Resources/views/ProductVariant/Update/_breadcrumb.html.twig]: Please use the proper route for this url: `sylius_admin_product_update`

**Comment by GSadee** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "variant-breadcrumb-link-product"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

File: src/Sylius/Bundle/AdminBundle/Resources/views/ProductVariant/Update/_breadcrumb.html.twig (hunk 1 above)
Comment (Tomanhez) [reply src/Sylius/Bundle/AdminBundle/Resources/views/ProductVariant/Update/_breadcrumb.html.twig]: Why? We need create link that it moves us to product index page not product edit page

Comment (GSadee) [reply src/Sylius/Bundle/AdminBundle/Resources/views/ProductVariant/Update/_breadcrumb.html.twig]: So.. should it moves us to product index page after clicking a **product id**?

And what is more, your implementation moves us to product show page, what you show us in screenschots, so I do not understand your comment..

Comment (GSadee) [reply src/Sylius/Bundle/AdminBundle/Resources/views/ProductVariant/Update/_breadcrumb.html.twig]: After clicking a product id on a product variant edit page, we should be moved to `sylius_admin_product_update` page

**Comment by GSadee** [comment]
Thank you, Tomasz! :1st_place_medal:

**Pull request merged**